packages/desktop-client/src/components/accounts/Balance.tsx

```diff
@@ -126,34 +148,58 @@ function FilteredBalance({ filteredAmount }) {
   );
 }
 
-function MoreBalances({ balanceQuery }) {
+function MoreBalances({
+  balanceQuery,
+}: {
+  balanceQuery: ComponentProps<typeof ReconcilingMessage>['balanceQuery'];
```

Comment:
Please extract to a Props type and this should be a `Binding`

Reply:
What makes you say that this should be a `Binding`? Do you have a suggested edit? 

I am seeing this in a different file, so I updated it to that for now.

https://github.com/actualbudget/actual/blob/339fac280605bbd344ecb3387751a9ee43ffab69/packages/desktop-client/src/components/accounts/Reconcile.tsx#L20